README.md

```diff
@@ -145,7 +145,7 @@ funcsave gdoc
 funcsave gtst
 ```
 
-If you prefer, you can paste these aliases directly in your `~/.gitconfig` file:
+If you prefer, you can paste these aliases directly in your `~/.gitconfig` file by using the command `notepad C:/Users/{name}/.gitconfig`, it will open a notepad editor for you containing git configurations.
```

Comment:
This is a universal repo, description regarding Windows is unnecessary. You can put it all up in an issue and link it there.